package.json

```diff
@@ -88,5 +90,9 @@
     "redux-logger": "^3.0.6",
     "redux-promise-middleware": "^5.1.1",
     "redux-thunk": "^2.3.0"
+  },
+  "jest": {
+    "snapshotSerializers": ["enzyme-to-json/serializer"],
```

Comment:
For jest configurations, you can add them to the `jest.config.json` file on the root folder. Reason is not to have a lot of jest config on the `package.json` file.

Reply:
done